vhpidirect/arrays/intvector/csized/makefile

```diff
@@ -0,0 +1,56 @@
+CC=gcc
+GHDL=ghdl
+
+CFLAGS= -fPIC
+
+STD=93
+
+WORK_DIR=.
```

Comment:
Since you already added this feature, the default might be `WORK_DIR?=wrk`. This would make `clean_work` slightly easier.

Reply:
Yup. only thing is that the work directory must exist. I'll look into how to get make to create the directory.